<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestaoFotos</class>
 <widget class="QDialog" name="GestaoFotos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestão de Fotos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitulo">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gestão de Fotos - Pintura:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBotoes">
     <item>
      <widget class="QPushButton" name="btnAdicionarFoto">
       <property name="text">
        <string>Adicionar Foto</string>
       </property>
       <property name="minimumHeight">
        <number>35</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoverFoto">
       <property name="text">
        <string>Remover Foto</string>
       </property>
       <property name="minimumHeight">
        <number>35</number>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditarDescricao">
       <property name="text">
        <string>Editar Descrição</string>
       </property>
       <property name="minimumHeight">
        <number>35</number>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAbrirPasta">
       <property name="text">
        <string>Abrir Pasta</string>
       </property>
       <property name="minimumHeight">
        <number>35</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widgetLista">
      <layout class="QVBoxLayout" name="layoutLista">
       <item>
        <widget class="QLabel" name="lblLista">
         <property name="text">
          <string>Fotos da Pintura:</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listaFotos">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetVisualizacao">
      <layout class="QVBoxLayout" name="layoutVisualizacao">
       <item>
        <widget class="QLabel" name="lblVisualizacao">
         <property name="text">
          <string>Visualização:</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>69</width>
            <height>69</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="layoutScroll">
           <item>
            <widget class="QLabel" name="lblImagem">
             <property name="text">
              <string>Selecione uma foto para visualizar</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>300</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblDescricao">
             <property name="text">
              <string>Descrição: </string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GestaoFotos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GestaoFotos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
